heroku/cli
# add features back to space creation

[GUS WI](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE000023YtrPYAS/view)

### Description

This adds the internal `--features` flag back to space creation now that `3.sdk` supports it.

### Testing
1. Pull down branch and `yarn build`
2. Run `./bin/run spaces:create SPACE_NAME --team heroku-dev-tools`
3. See success
